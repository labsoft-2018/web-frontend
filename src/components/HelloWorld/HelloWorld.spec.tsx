import * as React from 'react'
import { HelloWorld } from './'
import * as renderer from 'react-test-renderer'

test('HelloWorld', () => {
  const component = renderer.create(
    <HelloWorld name='Rafael' />,
  )

  expect(component.toJSON()).toMatchSnapshot()
})
