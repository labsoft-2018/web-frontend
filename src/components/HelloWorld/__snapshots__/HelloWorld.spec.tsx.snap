// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld 1`] = `
<h1>
  Hello, 
  Rafael
  !
</h1>
`;
